packages/js/README.md

```diff
@@ -238,6 +238,39 @@ use `client.getIsRegistered` method
 client.getIsRegistered().then(isRegistered => {...})
 ```
 
+### Anonymous Login for AI assistant
+
+The TelnyxRTC client can be used with an AI assistant by using the `anonymous_login` method. This allows you to authenticate without a SIP connection, making it easier to integrate with AI assistants.
```

Comment:
could you update the "quickstart" links in the docs to point to https://developers.telnyx.com/docs/voice/webrtc/sdk-commonalities#authentication?

Reply:
as far as I understood from readme, it's recommended to use JWT and/or to make sure to treat connection credentials as sensitive data. Is there a way to make this `anonymous_login` method won't necessary disclose the `target_id` in browser?